docs/guides/advanced/embeds.md

```diff
@@ -0,0 +1,71 @@
+# Embedding content
+
+Packs typically operate purely on data and actions, leaving the display up to the document. However there are times in which you may want to provide a richer experience, which can be accomplished by embedding a portion of an application directly within the doc.
+
+Coda uses the 3rd party service [Iframely][iframely] to handle embeds. Many popular apps and websites area already supported, and you can use their site to [check if a URL already supports embedding][iframely_check].
+
+
+## Adding support
+
+If you want to add embed support to your own application follow the instructions in the [Iframely documentation][iframely_docs]. This is most commonly done by adding support for the [oEmbed specification][oembed], which is used by many platforms. Once that is complete, [submit your application][iframely_submit] to Iframely to have it added to their registry.
+
+!!! note "User embeds"
+    Users can [embed content in their docs][help_center_embed] directly, without Packs, using the `/embed` slash command or the `=Embed()` formula. Even if you aren't planning to build a Pack you may want to support embeds for your users that manually embed the content in their docs.
+
+
+## Generating an embed
+
+Pack formulas can generate an embed by returning a URL with the `Embed` value hint.
+
+```ts
+pack.addFormula({
+  // ...
+  resultType: coda.ValueType.String,
+  codaType: coda.ValueHintType.Embed,
+  execute: async function ([], context) {
+    // TODO: Build or fetch embed URL.
+    return embedUrl;
+  },
+});
+```
+
+### Forcing an embed
+
+You may still be able to embed URLs that aren't supported by Iframely by using the `force` option.
+
+```ts
+pack.addFormula({
+  // ...
+  resultType: coda.ValueType.String,
+  schema: {
+    type: coda.ValueType.String,
+    codaType: coda.ValueHintType.Embed,
+    force: true,
+  },
+  execute: async function ([], context) {
+    // TODO: Build or fetch embed URL.
+    return embedUrl;
+  },
+});
+```
+
+A forced embed is initially displayed in a disabled state:
```

Comment:
maybe note that this is a once per domain / per user disabled state (i.e. once you approve that domain you will not have to approve again in future, similarly if you approve it does not mean others will not see it)

Reply:
Great callout! I meant to include that and forgot, and also didn't actually know the scope. Done.